app/build.gradle

```diff
@@ -226,9 +246,9 @@ def addSecretToBuildConfig(variant, fieldName, fileBaseName) {
     // We separate the debug files from the release files so that we don't
     // accidentally ship a release using the debug key or vice versa.
     def fileSuffix
-    if (variantName.contains("debug")) {
+    if (variantName.contains("Debug")) {
         fileSuffix = "debug"
-    } else if (variantName.contains("release")) {
+    } else if (variantName.contains("Release")) {
```

Comment:
Optional nit: a potentially less brittle way of handling this is `variantName.toLowerCase().contains("release")`